CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+## Contributing
+
+1. Familiarize yourself with the codebase by reading the [docs](docs).
+1. Create a new issue before starting your project so that we can keep track of
+   what you are trying to add/fix. That way, we can also offer suggestions or
+   let you know if there is already an effort in progress.
+1. Fork this repository.
+1. The [README](README.md) has details on how to set up your environment.
+1. Create a _topic_ branch in your fork based on the correct branch (usually the **develop** branch, see [Branches section](#branches) below). Note, this step is recommended but technically not required if contributing using a fork.
+1. Edit the code in your fork.
+1. Sign CLA (see [CLA](#cla) below)
+1. Send us a pull request when you are done. We'll review your code, suggest any
+   needed changes, and merge it in.
+
+### Committing
+
+1. We enforce commit message format. We recommend using [commitizen](https://github.com/commitizen/cz-cli) by installing it with `npm install -g commitizen` and running `npm run commit-init`. When you commit, we recommend that you use `npm run commit`, which prompts you with a series of questions to format the commit message. Or you can use our VS Code Task `Commit`.
```

Comment:
It doesn't seem like scripts are included in the repo for commit or commit-init - should this section be removed, or do we want to add those scripts to the repo?

Reply:
Good catch.  I thought commitizen was working but it wasn't.  (this file was mostly copy-pasted from our extension repo)  I've added the script and added the commitizen dev dependencies.